### PR TITLE
REST-64: Add PHP GitHub CI setup documentation

### PR DESCRIPTION
**Pull request by LakshayaT**
Task ID: REST-64

This PR adds the Github CI setup documentation for PHP projects.

Additionally, this PR also adds the `assets/` subfolder which will be used for holding files and images as required.

### CODE REVIEW

File: ci-setups/php-github.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+    ![Github New workflow](assets/php-github-new-workflow.png)
+
+4. Add the following contents to this file:
+
+    ```yaml
+    name: PHP Coding Standards
+    on:
+      push:
+        branches: [ main ]
```
Comment (osm-vishnukyatannawar) [inline ci-setups/php-github.md:46]: Add example for multiple branches here.

**Review by osm-vishnukyatannawar — CHANGES_REQUESTED** [review]

File: ci-setups/php-github.md (hunk 1 above)
Comment (LakshayaT) [reply ci-setups/php-github.md:46]: This has been addressed in the most recent commit.

File: ci-setups/php-github.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
+# Continuous Integration (CI) Setup for PHP projects in Github
```
Comment (osm-vishnukyatannawar) [inline ci-setups/php-github.md:1]: Github  --> GitHub

Fix it elsewhere please.
Ensure it is handled for GitLab too.

**Review by osm-vishnukyatannawar — CHANGES_REQUESTED** [review]

File: ci-setups/php-github.md (hunk 2 above)
Comment (LakshayaT) [reply ci-setups/php-github.md]: All occurrences of Github in the document have been updated to GitHub.

Similar change has been made to occurrences of Gitlab -> GitLab.

**Review by osm-vishnukyatannawar — APPROVED** [review]

**Pull request merged**